source/ore/publish.rst

```diff
@@ -145,3 +145,57 @@ Support
 -------
 
 Support cannot do anything, merely a way of showing that the member is a part of the project.
+
+
+Documenting Your Project With Pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To document your plugin Ore offers the creation of pages, by default you get a 'Home' page when you create your new Project.
+To add new pages you can click on the '+' icon in the Pages bar on the right of the screen, doing so will open a pop-up where you 
```

Comment:
Again, please replace the comma. A semicolon would do here "... of the screen; doing so will ..."